### ISSUE TITLE
Feature request - No Google - use iCloud instead

### ISSUE THREAD

**Issue by basinger** [dbaldwin/DroneBlocks-Web#16]
DroneBlocks uses Google. Google is blocked in China and therefore for users in China it's a big hurdle. It works with VPN but it still is a bit glitchy. 
Would be great if it would use iCloud instead - or maybe as an option (Google or iCloud). Dropbox is also blocked by the way.

**Comment by dbaldwin** [comment]
Are you referring to the login method? What if we introduced a Facebook login option?

**Comment by basinger** [comment]
Google for login is a problem. Facebook would be too - also blocked in China. Only accessible with a VPN. Another question after login would be how the files are stored. I assumed with a google login they are stored on some google server as well.

**Comment by dbaldwin** [comment]
@basinger do you know if VPN access in China is still accessible? I've read that even many VPN apps have been blocked. We are still looking how to overcome this.

**Comment by basinger** [comment]
Yes I know. I just came back to China after a short holiday and my VPN still works. I know that the government has e.g. requested Apple to remove all VPN apps from the Chinese Appstore. Only the ones where the government has access to are still allowed. But I doubt they are really able to block all VPN ever. It’s a cat and mouse game. But having to use VPN is a pain in the ass.


On 9 Oct 2017, at 04:15, Dennis Baldwin <notifications@github.com<mailto:notifications@github.com>> wrote:


@basinger<https://github.com/basinger> do you know if VPN access in China is still accessible? I've read that even many VPN apps have been blocked. We are still looking how to overcome this.

**Comment by dbaldwin** [comment]
Do you have a VPN client that you recommend? We a few users that have been asking so I'm trying to understand what we can do the interim. Thank you.

**Comment by basinger** [comment]
I am happy with Astrill. ExpressVPN is also very common here in China. No service works always though. There are sometimes blackouts.

On 9 Oct 2017, at 21:17, Dennis Baldwin <notifications@github.com<mailto:notifications@github.com>> wrote:


Do you have a VPN client that you recommend? We a few users that have been asking so I'm trying to understand what we can do the interim. Thank you.
